### PR TITLE
Separate the sidebars out

### PR DESCRIPTION
**Pull request by olijeffers0n**
Separates the sidebars for the projects for admin/dev/contrib.

I would also like to make it so that search results only show for the correct project. LMK if you know how

### CODE REVIEW

**Comment by lynxplay** [comment]
The project level separation I definitely agree with, separating the top levels of each project too however might be too much IMO. Tho, not a hard no on that, would need more opinions.

**Comment by olijeffers0n** [comment]
You mean like hiding the other projects at the top when you select one?

**Comment by lynxplay** [comment]
No, I mean that https://docs-k2bymnn76-papermc.vercel.app/paper/dev does not show administration or contributing in the sidebar.

File: docs/paper/README.mdx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "href": "/paper/admin",
+  },
+  {
+    "type": "link",
+    "label": "Paper Development Guide",
+    "href": "/paper/dev",
+  },
+  {
+    "type": "link",
+    "label": "Paper Contribution Guide",
```
Comment (powercasgamer) [inline docs/paper/README.mdx:22]:
```suggestion
    "label": "Paper Contributing Guide",
```
match title here <https://docs-k2bymnn76-papermc.vercel.app/paper/contributing>

File: config/sidebar.velocity.ts [hunk 2]
```diff
@@ -1,9 +1,8 @@
 import type { SidebarsConfig } from "@docusaurus/plugin-content-docs";
 
 const velocity: SidebarsConfig = {
-  primary: [
-    "README",
-    {
+  primary: ["README",],
```
Comment (powercasgamer) [inline config/sidebar.velocity.ts:4]:
```suggestion
  primary: ["README"],
```
looks dum

**Review by powercasgamer — CHANGES_REQUESTED** [review]

File: config/sidebar.paper.ts [hunk 3]
```diff
@@ -3,174 +3,177 @@ import type { SidebarsConfig } from "@docusaurus/plugin-content-docs";
...
+      type: "doc",
+      id: "admin/README",
+    },
+    items: [
+      {
+        type: "category",
+        label: "Getting Started",
+        link: {
+          type: "generated-index",
+          slug: "/cat/admin/getting-started",
```
Comment (powercasgamer) [inline config/sidebar.paper.ts:24]: Why is `cat` here? The development docs don't have it

Comment (olijeffers0n) [reply config/sidebar.paper.ts:24]: Cause the docs have horribly enforced standards :P

**Review by powercasgamer — APPROVED** [review]

File: config/sidebar.paper.ts [hunk 4]
```diff
@@ -3,173 +3,179 @@ import type { SidebarsConfig } from "@docusaurus/plugin-content-docs";
...
-      label: "Administration",
-      collapsed: true,
-      link: {
-        type: "doc",
-        id: "admin/README",
+    "admin/README",
+    "dev/README",
+    "contributing/README",
+  ],
+  administration: [{
```
Comment (powercasgamer) [inline config/sidebar.paper.ts:10]:
```suggestion
  administration: [
  {
```

File: config/sidebar.paper.ts [hunk 5]
```diff
@@ -3,173 +3,179 @@ import type { SidebarsConfig } from "@docusaurus/plugin-content-docs";
...
-      ],
+        items: [
+          "admin/misc/faq",
+          "admin/misc/paper-bug-fixes",
+        ],
+      },
+    ],
+  },
+  ],
+  development: [{
```
Comment (powercasgamer) [inline config/sidebar.paper.ts:88]:
```suggestion
  development: [
  {
```

File: config/sidebar.paper.ts [hunk 6]
```diff
@@ -3,173 +3,179 @@ import type { SidebarsConfig } from "@docusaurus/plugin-content-docs";
...
+          "dev/misc/internal-code",
+        ],
       },
-      items: [
-        "contributing/events",
-      ],
+    ],
+  },
+  ],
+  contributing: [{
```
Comment (powercasgamer) [inline config/sidebar.paper.ts:167]:
```suggestion
  contributing: [
  {
```

File: config/sidebar.velocity.ts [hunk 7]
```diff
@@ -3,7 +3,8 @@ import type { SidebarsConfig } from "@docusaurus/plugin-content-docs";
 const velocity: SidebarsConfig = {
   primary: [
     "README",
-    {
+  ],
+  administration: [{
```
Comment (powercasgamer) [inline config/sidebar.velocity.ts:7]:
```suggestion
  administration: [
  {
```

File: config/sidebar.velocity.ts [hunk 8]
```diff
@@ -55,7 +56,8 @@ const velocity: SidebarsConfig = {
         },
       ],
     },
-    {
+  ],
+  development: [{
```
Comment (powercasgamer) [inline config/sidebar.velocity.ts:60]:
```suggestion
  development: [
  {
```

**Review by powercasgamer — CHANGES_REQUESTED** [review]

**Comment by olijeffers0n** [comment]
So I have updated this. I made it off the back of the comments from community members however I am not sure how much this really improves the UX - let me know what you think

**Comment by olijeffers0n** [comment]
Looking at this, i'm not a fan of how this works.

**Pull request closed without merging**